packages/valory/services/oracle_goerli/service.yaml

```diff
@@ -61,9 +59,9 @@ type: skill
         tendermint_p2p_url: ${TM_P2P_ENDPOINT_NODE_0:str:node0:26656}
         reset_pause_duration: 3000
         broadcast_to_server: true
-        share_tm_config_on_startup: ${USE_ACN:bool:false}
+        service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a}
+        share_tm_config_on_startup: ${SHARE_TM_CONFIG_ON_STARTUP:bool:false}
```

Comment:
Propel needs this set to `true`, correct? If yes, then let's set it as the default value as it makes more sense.

Reply:
This only true when deploying from `from-token` command. On propel we can just set env var `SHARE_TM_CONFIG_ON_STARTUP` to true

Reply:
So then it makes more sense to be `true` by default.

Reply:
Agree with @Adamantios , because acn is above also set to true, the two should be in synchrony

Reply:
Addressed [46d5a59](https://github.com/valory-xyz/price-oracle/pull/59/commits/46d5a59e90f0d98291fe6f78ffc456cdf07715c2)